src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import * as S from "./Navbar.Styles"
+import logo from "../../assets/logo.png"
+import profile from "../../assets/profile.png"
+import Button from "../Button/Button"
+import { useNavigate } from "react-router-dom"
+import SearchIcon from "@mui/icons-material/Search"
+
+const navMenu = ["게시물 생성", "DM", "알림", "로그인"]
+
+const Navbar = () => {
+  const navigate = useNavigate()
+  return (
+    <S.NavbarLayout>
+      <S.NavbarLeftList>
+        <S.NavbarButton onClick={() => navigate("/")}>
+          <S.NavbarLogo
+            src={logo}
+            alt="로고"
+          />
+        </S.NavbarButton>
+        {/* 나중에 SearchModal과 연결 */}
+        <S.NavbarButton>
+          <SearchIcon />
+        </S.NavbarButton>
+      </S.NavbarLeftList>
+      <S.NavbarRightList>
+        {navMenu.map((menu) => (
+          <Button
+            height="4.5rem"
+            fontSize="2rem"
+          >
+            {menu}
+          </Button>
+        ))}
+
+        <S.NavbarButton>
+          <S.NavbarProfile
+            src={profile}
+            alt="프로필"
+          />
+        </S.NavbarButton>
+      </S.NavbarRightList>
+    </S.NavbarLayout>
+  )
+}
```

Comment:
```suggestion
  return (
    <S.NavbarLayout>
      <NavbarLeftList />
      <NavbarRightList />
    </S.NavbarLayout>
  )
}
```

[ P2 ] 위와 같이 별도의  Component로 관리하는건 어떨까요?
일단 JSX 문법이 많다보니 가독성에 좋지 않아 분리하시면 좋을 것 같아요.
그리고 분리하면 이후 작업할 버튼의 기능에 대해서도 각각의 Component로 관리할 수 있어 유지보수에서도 큰 이점이라고 생각되어지네요!
추가로 해당 Component는 이후 Nav에서 전역으로 상태 관리를 위한 공간으로 활용될 수 있겠네요!